src/stylus/components/_pagination.styl

```diff
@@ -37,12 +37,12 @@
     text-decoration: none
     
     &--active
-      @extend .z-depth-2
+      elevation(4)
```

Comment:
@mfferreira I really like the elevation on active. shall we maybe introduce the same elevation(4) to hover?